src/js/components/List/List.js

```diff
@@ -52,6 +53,17 @@ const StyledItem = styled(Box)`
   &:focus {
     ${unfocusStyle({ forceOutline: true, skipSvgChildren: true })}
   }
+  ${(props) => {
+    let disabledStyle;
```

Comment:
Wondering if we should have a hover over if it is `disabled` on DataTable the `disabled` story the hover behavior is not there http://localhost:9001/?path=/story/visualizations-datatable-onselect--on-select-data-table

Reply:
Mmmm... good call. I think we want the active styling when using keyboard (because I need to know which item I am currently on), but agree that it is unnecessary when mouseover.

Reply:
I agree with the using `Keyboard` 👍

Reply:
addressed.

Reply:
nice much better!